source/codegen/common_helpers.py

```diff
@@ -495,7 +495,10 @@ def get_twist_value(parameters):
     for p in parameters:
         if is_array(p['type']):
             size = p.get('size', {})
-            return size.get('value_twist', None)
+            value_twist = size.get('value_twist', None)
```

Comment:
This function was looking for the first array in the list of parameters and assuming that was the input that had a `value_twist`. Now we only return a non-None `value_twist` (if there is one).